OCPBUGS-7229: OpenStack: Add load balancer Services limitations (#56004)
openshift/openshift-docs: modules/nw-osp-loadbalancer-limitations.adoc

// Module included in the following assemblies:
// * networking/load-balancing-openstack.adoc
// For thinking and reviewing, adding to networking/load-balancing-openstack.adoc

:_content-type: CONCEPT
[id="nw-osp-loadbalancer-limitations_{context}"]
= Limitations of load balancer services

{product-title} clusters on {rh-openstack-first} use Octavia to handle load balancer services. As a result of this choice, such clusters have a number of functional limitations.

> {rh-openstack} Octavia has two available providers: Amphora and OVN. These providers differ in terms of available features as well as implementation details. These distinctions affect load balancer services created on your cluster.

Reviewer: ```suggestion
{rh-openstack} Octavia has two supported providers: Amphora and OVN. These providers differ in terms of available features as well as implementation details. These distinctions affect load balancer services that are created on your cluster.
```

> include::modules/nw-osp-loadbalancer-etp-local.adoc[leveloffset=+1]

Reviewer 2: I think maybe this is the reason that the HTML preview isn't working - it's my understanding that you can't include a module in another module, as is being done with the include on this line and the include below. I think you would need to go up to the assembly and include all of the modules there, using the leveloffset to create the different heading levels.

Reviewer: @dulek This is correct. Can we pull this statement into the assembly?

// Module included in the following assemblies:
// * networking/load-balancing-openstack.adoc
// For thinking and reviewing, adding to networking/load-balancing-openstack.adoc

:_content-type: CONCEPT
[id="nw-osp-loadbalancer-limitations_{context}"]

> = Limitations of load balancer Services

Reviewer: ```suggestion
= Limitations of load balancer services
```

> {product-title} clusters on {rh-openstack-first} use Octavia to handle load balancer Services. You should be aware of some functional limitations that are a result of this.

Reviewer: ```suggestion
{product-title} clusters on {rh-openstack-first} use Octavia to handle load balancer services. As a result of this choice, such clusters have a number of functional limitations.
```

> {rh-openstack} Octavia has two available providers: Amphora and OVN. These two differ in terms of available features as well as implementation details and these distinctions affect load balancer Services created in {product-title}.

Reviewer: ```suggestion
{rh-openstack} Octavia has two available providers: Amphora and OVN. These providers differ in terms of available features as well as implementation details. These distinctions affect load balancer services created on your cluster.
```

> == `externalTrafficPolicy` set to `Local`

Reviewer: It's a pain, but these two H2s will probably have to be broken into their own discrete modules to adhere to our mod docs style.

They could also become an unordered list, though the first section might be a bit long for that.

Do you have a preference for either approach as I review this? @dulek 

Pausing here.

Author: I don't have a preference, let me break it down.
